### ISSUE TITLE
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an al...

### ISSUE THREAD

**Issue by sebj54** [1000ch/atom-svgo#4]
Hi,

I got this error message with a fresh install of the svgo plugin, hope you can help.

**Atom Version**: 1.7.1
**System**: Ubuntu 15.10
**Thrown From**: [svgo](https://github.com/1000ch/atom-svgo) package, v1.2.5

### Stack Trace

Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

```
At /home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/csso/lib/compressor/index.js:74

EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

    at Function (native)
    at compress (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/csso/lib/compressor/index.js:74:74)
    at minify (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/csso/lib/index.js:95:9)
    at Object.minifyBlock (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/csso/lib/index.js:120:12)
    at Object.exports.fn (/home/sebj/.atom/packages/svgo/node_modules/svgo/plugins/minifyStyles.js:38:42)
    at /home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/plugins.js:59:45
    at Array.filter (native)
    at monkeys (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/plugins.js:46:39)
    at perItem (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/plugins.js:77:12)
    at /home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/plugins.js:18:24
    at Array.forEach (native)
    at module.exports (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/plugins.js:14:13)
    at /home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo.js:63:17
    at Object.sax.onend (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/svg2js.js:156:13)
    at emit (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/sax/lib/sax.js:640:35)
    at end (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/sax/lib/sax.js:683:5)
    at Object.write (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/sax/lib/sax.js:991:14)
    at Object.SAXParser.close (/home/sebj/.atom/packages/svgo/node_modules/svgo/node_modules/sax/lib/sax.js:157:38)
    at module.exports (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo/svg2js.js:169:28)
    at [object Object].SVGO._optimizeOnce (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo.js:56:5)
    at [object Object].SVGO.optimize (/home/sebj/.atom/packages/svgo/node_modules/svgo/lib/svgo.js:49:11)
    at minify (/home/sebj/.atom/packages/svgo/index.js:27:10)
    at atom-workspace.<anonymous> (/home/sebj/.atom/packages/svgo/index.js:37:60)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:260:29)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:00.9.0 svgo:minify (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "open-on-search-engine",
      "symbols-view",
      "lmgtfy",
      "lmgtfy"
    ],
    "packagesWithKeymapsDisabled": [
      "svgo"
    ],
    "themes": [
      "soft-flat-ui",
      "soft-flat-ui-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-minify, v0.7.1
emmet, v2.4.3
file-types, v0.5.1
highlight-selected, v0.11.2
minimap, v4.22.1
release-notes, v0.53.0
remote-sync, v4.0.1
set-syntax, v0.3.0
soft-flat-ui, v0.1.0
soft-flat-ui-syntax, v0.1.0
svgo, v1.2.5
sync-settings, v0.7.2
trailing-spaces, v0.3.2
webbox-color, v0.5.9

# Dev
No dev packages
```

**Comment by 1000ch** [comment]
@sebj54 Thank you for reporting. I fixed this, so try `v2.0.0`?

**Comment by sebj54** [comment]
Thank you for the quick fix, it works great now!
